README.md

```diff
@@ -1,21 +1,63 @@
-# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 Olympick app 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
+# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 🇯🇵 Olympick 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-Currently there is no way to create a viewing schedule for the Olympics that only includes specific sports.
+Currently there is no way to create a viewing schedule for the Olympics that only include specific sports. Olympick aims to solve this problem by helping people create their own personalised schedule for watching events of their favourite sports in UK time. Events already watched or no longer of interest can then be removed from schedules and/or new events can be added.
 
```

Comment:
Could you please add the image I sent last night to the group as well :) 
Thanks